var/spack/repos/builtin/packages/flex/package.py

```diff
@@ -51,7 +51,9 @@ class Flex(AutotoolsPackage):
 
     def url_for_version(self, version):
         url = "https://github.com/westes/flex"
-        if version >= Version('2.6.1'):
+        if version >= Version('2.6.3'):
+            url += "/releases/download/v{0}/flex-{0}.tar.gz".format(version)
+        elif version >= Version('2.6.1'):
```

Comment:
Related to comment above about issues with resolving 2.6.3, I'll drop this line.